tests/brun/max-cost-1.txt

```diff
@@ -1,3 +1,3 @@
-brun -c -m 75 '((c (q . 1) (q . (100 200))))'
-cost = 75
-(100 200)
+brun -c -m 73 '(c (q . 100) (q . 200))'
```

Comment:
I think you should add a test to see `((c ...` fail

Reply:
Done.

Reply:
(Except it doesn't yet fail, that comes after a spin of clvm_tools.)